components/Polls/PollList.js

```diff
@@ -0,0 +1,165 @@
+import React, { Component } from 'react';
+import { ListView, View, TextInput, TouchableOpacity, NavigatorIOS } from 'react-native';
+import { Container, Content, Text, Button, ListItem, Icon } from 'native-base';
+import { firebaseDB } from '../../firebase/firebaseHelpers';
+import Poll from './Poll';
+
+export default class PollList extends Component {
+  //input new poll name
+  //submit button to create a new poll entry in db
+    //anncounce it in chat
+    //forward to Poll component
+
+  //show list of active polls
+  //allow poll deletion
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      username: this.props.user ? this.props.user.displayName : 'Anonymous',
+      group: this.props.groupName ? this.props.groupName : 'Default',
+      input: '',
+      polls: [],
+    };
+    this.ds = new ListView.DataSource({rowHasChanged: (r1, r2) => r1 !== r2});
+    this.addPoll = this.addPoll.bind(this);
+    this.removePoll = this.removePoll.bind(this);
+  }
+
+  componentWillMount() {
+    this.getPolls();
+  }
+
+  getPolls() {
```

Comment:
Could these go into the firebaseHelpers?  You are probably already refactoring this, but just a thought.